renku_notebooks/api/classes/server.py

```diff
@@ -727,6 +727,23 @@ def _get_session_manifest(self):
                     ],
                 }
             )
+        patches.append(
+            {
+                "type": "application/json-patch+json",
+                "patch": [
+                    {
+                        "op": "add",
+                        "path": "/ingress/metadata/annotations/-",
```

Comment:
Isn't that the syntax for adding to an array? The annotations are a map iirc.

Reply:
Ah yeah that is true. My bad.